backend/src/main/java/botobo/core/application/WorkbookService.java

```diff
@@ -56,4 +87,12 @@ public WorkbookCardResponse findWorkbookCardsById(Long id) {
                 .orElseThrow(WorkbookNotFoundException::new);
         return WorkbookCardResponse.of(workbook);
     }
+
+    @Transactional
+    public void deleteWorkbook(Long id, AppUser appUser) {
+        Workbook workbook = workbookRepository.findById(id)
+                .orElseThrow(WorkbookNotFoundException::new);
+        workbook.deleteIfCan(appUser.getId());
+        workbookRepository.delete(workbook);
+    }
```

Comment:
```suggestion
    public void deleteWorkbook(Long id, AppUser appUser) {
        Workbook workbook = workbookRepository.findById(id)
                .orElseThrow(WorkbookNotFoundException::new);
        workbook.isOwner(appUser.getId());
        workbookRepository.delete(workbook);
    }
```
workbook 을 지울 때 꼭 user 의 workbooks 에서 해당 workbook 을 제거할 필요는 없다고 저는 생각해요! 어차피 지워지니까!

Reply:
사실 그렇긴해요ㅋㅋㅋㅋ 요거도 좀만 생각해보고 수정할게요!!